src/ApiRequestPanel.js

```diff
@@ -447,6 +475,20 @@ export class ApiRequestPanel extends AmfHelperMixin(EventsTargetMixin(HeadersPar
     this.requestUpdate('selectedServerValue', old);
   }
 
+  get noServerSelector() {
+    return this._noServerSelector;
+  }
+
+  set noServerSelector(value) {
+    const old = this.noServerSelector;
+    if (old === value) {
+      return;
+    }
+
+    this._noServerSelector = value;
+    this._computeServerSelectorHidden();
```

Comment:
I know that this setter calls a function that in the end calls `requestUpdate()` but please put it in here too. This property is used when the template is generated so it must be reactive. When you remove `_computeServerSelectorHidden()` function call here you won't track back that this means that this won't call for render.